demos/graph-classification/supervised-graph-classification.ipynb

```diff
@@ -93,7 +93,8 @@
     "from tensorflow.keras.layers import Dense\n",
```

Comment:
<p>Maybe instead of having to update the 75.9 ± 6.3 number here every time the notebook is rerun (since this notebook is not reproducible), we could not be quite so specific, and just refer back to the place that computes the number. The paragraph starting with "Given the above ... " could be changed to something like</p><p> </p><blockquote>The performance results reported above likely have high variance because of the small dataset size.</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1387/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>